### PR TITLE
Feature/day 11

### PR DESCRIPTION
**Pull request by xwang1-1**

### CODE REVIEW

File: d011-blackjack/Untitled-1.py
Comment (amoralesch) [inline d011-blackjack/Untitled-1.py:1]: Is there a better name for the file?

File: d011-blackjack/Untitled-1.py [hunk 1]
```diff
@@ -0,0 +1,128 @@
+#initially, randomly distribute two cards each to house and player
```
Comment (amoralesch) [inline d011-blackjack/Untitled-1.py]: Add a little documentation:

- Name of the project/game
- What is the purpose?
- What did you learn from it?

File: d011-blackjack/Untitled-1.py
Comment (amoralesch) [inline d011-blackjack/Untitled-1.py:1]: If you open the file with PyCharm, is going to show some warnings about code styles, can we fix those before merging the code?

File: d011-blackjack/Untitled-1.py [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+         8,
+         9,
+        10,
+       'J',
+       'Q',
+       'K']
+    
+house_hand = []
+player_hand = []
+#random.shuffle(card)
```
Comment (amoralesch) [inline d011-blackjack/Untitled-1.py]: Let's remove commented out code from the file.

File: d011-blackjack/Untitled-1.py [hunk 3]
```diff
@@ -0,0 +1,128 @@
...
+house_hand.append(random.choice(card))
+player_hand.append(random.choice(card))
+#print(house_hand)
+#print(player_hand)
+#print(player_hand[0])
+
+house_value = 0
+player_value = 0
+
+def Card_Value(card):
```
Comment (amoralesch) [inline d011-blackjack/Untitled-1.py]: There are many ways to name variables and functions. In python, the convention is _snake_case_, i.e. all words in lowercase, with underscores (`_`) as separator.

File: d011-blackjack/Untitled-1.py [hunk 4]
```diff
@@ -0,0 +1,128 @@
...
+    house_value = TotalValue(house_hand)
+    #house_value += Card_Value(house_hand[len(house_hand)-1])
+    #if house_hand[len(house_hand)-1] == 'A' and house_value > 21:
+     #   house_value -= 10
+    print(f"House: {house_hand} | {house_value}")
+
+    
+    
+    
+print("-------------Game result-------------------")
```
Comment (amoralesch) [inline d011-blackjack/Untitled-1.py]: Lots of these _prints_ looks similar, can we simplify them somehow?

File: d011-blackjack/Untitled-1.py [hunk 5]
```diff
@@ -0,0 +1,128 @@
...
+        sum_value -=10
+        ace_count -=1
+    return sum_value
+
+
+#house_value = Card_Value(house_hand[0]) + Card_Value(house_hand[len(house_hand)-1])
+#player_value = Card_Value(player_hand[0]) + Card_Value(player_hand[len(player_hand)-1])
+player_value = TotalValue(player_hand)
+house_value = TotalValue(house_hand)
+print(f"Player: {player_hand} | {player_value}")
```
Comment (amoralesch) [inline d011-blackjack/Untitled-1.py]: In a normal game, the user can see _one_ card from the house before deciding if he/she wants more cards. Can we add the same option here?

**Review by amoralesch — CHANGES_REQUESTED** [review]
Some comments added. We can work on them together if you prefer.

File: d011-blackjack/Untitled-1.py (hunk 1 above)
Comment (xwang1-1) [reply d011-blackjack/Untitled-1.py]: - Blackjack game, playing cards between the House and the Player. 
- Random function was used in the game
- A list is capable for both integers and string values

File: d011-blackjack/Untitled-1.py (hunk 2 above)
Comment (xwang1-1) [reply d011-blackjack/Untitled-1.py]: Comment removed.

File: d011-blackjack/Untitled-1.py (hunk 1 above)
Comment (amoralesch) [reply d011-blackjack/Untitled-1.py]: I added the same (almost exactly) the the file

**Review by amoralesch — APPROVED** [review]

**Pull request merged**